Cdn_BunnyCdn_Widget_View_Unauthorized.php

```diff
@@ -13,7 +14,105 @@
 ?>
 
 <div id="bunnycdn-widget" class="w3tc_bunnycdn_signup">
-	<?php if ( ! $c->get_boolean( 'cdn.enabled' ) ) : ?>
+	<?php
+	$cdn_engine = $c->get_string( 'cdn.engine' );
```

Comment:
The site may have Bunny CDN configured as full-site delivery, so check both settings (enabled and engine) to see and only display the notice if not already using Bunny for one of them.

Reply:
Updated to take into account CDN enabled and engine selection in case of bunny